libexec/rc/rc.subr

```diff
@@ -2061,6 +2061,24 @@ find_local_scripts_old() {
 	done
 }
 
+_add_local_script()
+{
+	local file prefered service
+	file="$1"
+	service=${file##*/}
+	if [ -x "/etc/rc.d/${service}" ]; then
```

Comment:
This choice is interesting and might be the use case to consider upstreaming.  I suspect that actually users might want to prefer /usr/local/etc/rc.d/foo over /etc/rc.d/foo so that they can do overrides?

Reply:
My original version just ignored /etc/rc.d entirely. Allowing overrides seems like a potential source of confusion and it would be incompatible with service(8). We could record services that have run in the "early" stage and allow ones in the late stage to be overridden.